tests/functional_tests/t0_main/debug/t5_sentry_third_party.yea

```diff
@@ -6,7 +6,7 @@ tag:
     - linux
     - mac
 command:
-  program: sentry-third-party.py
+  program: sentry_third_party.py
```

Comment:
`tests/functional_tests/t0_main/debug/sentry-third-party.py` needs to be renamed

Reply:
fixed,. thanks